libclc/test/misc/convert.cl

```diff
@@ -0,0 +1,32 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// RUN: %clang -target amdgcn-mesa-mesa3d -O1 -emit-llvm -S -o - --libclc-lib=tahiti-amdgcn-mesa-mesa3d -fno-builtin --no-offloadlib %s | FileCheck %s
```

Comment:
Pretty sure you shouldn't need -fno-builtin or --no-offloadlib. Particularly -fno-builtin. OpenCL isn't really an offload language, but did that end up as the alias for nogpulib?

Reply:
Yes, `no-offloadlib` is the alias for `nogpulib`. I'll try without them but yeah if we're generating LLVM IR then it's not going to try and link in the libdevice, is it?

Reply:
It will be linked in, that's the point of the test?